.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Conventional Commits
+
+This repository follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#specification); please add your conventional commit message to this PR's description in the form:
+
+```text
+type[!]: description
```

Comment:
I think that this one should be part of the PR title, since the merge commit is picking the PR title as the first line of the commit message.